api/src/main/java/org/pmiops/workbench/cohortbuilder/SearchTerm.java

```diff
@@ -29,11 +29,7 @@ public String getTerm() {
 
   public String getCodeTerm() {
     // could just be a static method
-    if (term.startsWith("-")) {
-      return term.substring(1).replaceAll("[()+\"*]", "");
-    } else {
       return term.replaceAll("[()+\"*]", "");
-    }
```

Comment:
This change addresses item #3

Reply:
Don't know what is the right answer for this. But need broken tests fixed.